### DISCUSSION TITLE
Undocumented switch: `--os`

### DISCUSSION THREAD

**Discussion by tiefenauer** [gitbls/sdm#51] [Q&A] [answered]
The CLI exhibits a `--os` switch, which is not documented in the [Command Details](https://github.com/gitbls/sdm/wiki/Command-Details):

```bash
$ sdm --help
(...)
 --os osname         Specify OS in the Image (D:raspios, ubuntu)
(...)
```
Could you update the docs explaining what this switch is used for and what effect setting it will have?

**Comment by gitbls** [comment, marked as answer]
It's undocumented because it's currently unimplemented. It's a placeholder for some possible future work.
